ui/src/main.rs

```diff
@@ -434,6 +434,20 @@ fn use_app_coroutines(cx: &ScopeState) -> Option<()> {
         }
     });
 
+    //Update active call
+    use_future(cx, (), |_| {
+        to_owned![state];
+        async move {
+            loop {
+                sleep(Duration::from_secs(1)).await;
+                log::trace!("updating active call");
+                if state.write_silent().ui.call_info.update_active_call() {
```

Comment:
Here we need to use read(), right? 


```suggestion
                if state.read().ui.call_info.update_active_call() {
```

Reply:
`update_active_call` is a mutable function as outlined in [common/src/state/call.rs](https://github.com/Satellite-im/Uplink/pull/1207/files#diff-134fd3263262466455426cd37bd253a755ba88c9e9e1a201578de7b86c4ace62) so calling `read` would cause an error.